homeassistant/components/harmony/config_flow.py

```diff
@@ -14,7 +14,13 @@
 from homeassistant.const import CONF_HOST, CONF_NAME
 from homeassistant.core import callback
 
-from .const import ATTR_ACTIVITY_NOTIFY, DOMAIN, UNIQUE_ID
+from .const import (
+    ATTR_ACTIVITY_NOTIFY,
```

Comment:
I've been looking at the docs and the source, it's unclear to me, what does this option exactly do? Is it to include the activity? Why don't we always do that?

Reply:
If `ATTR_ACTIVITY_NOTIFY` is `True` the activity is updated right after you press the button on the remote instead of when the activity finishes setting up.  I almost want to remove the option and force it to `True` if the old default wasn't `False` for so many years because it means any automations or linked entities have to wait 30-40 seconds to actually trigger an action.  It also makes homekit confusing because the state changes to the new activity so far away from when you took the action.

Reply:
That seems like a weird option. Let's just make it True.

Reply:
Killing with fire